### ISSUE TITLE
Some array functions are missing in pg_proc.h

### ISSUE THREAD

**Issue by 0x0FFF** [greenplum-db/gpdb#482]
The functions like array_length and array_ndims are missing in pg_proc.h regardless they are implemented in the code

**Comment by ascherbaum-pivotal** [comment]
Merged.

**Comment by kuien** [comment]
From my investigation, most source-codes related to array functions are backported onto master except the cardinality(). It was first introduced in 
```
commit f98f6ee0641e87c6ecc2524f5d0a8b54924ffd14
Author: Peter Eisentraut <peter_e@gmx.net>
Date:   Wed Nov 12 13:09:28 2008 +0000
```
and later be refined in 
```
commit f2110a757d0bdca766fa1b8562c7dcb1a53fd422
Author: Tom Lane <tgl@sss.pgh.pa.us>
Date:   Sun Apr 5 22:28:59 2009 +0000
```

In this case, should we backport cardinality() as well to ensure the integrity of PG83 updating?

@andreasscherbaum @hlinnaka @danielgustafsson what're your thoughts?

**Comment by danielgustafsson** [comment]
Do we (or any user) need it? This is part of the 8.4 cycle AFAICT and will get merged when we work on syncing with the 8.4 upstream. I'm not debating it's usefulness but the same argument can be had for any commit from upstream so it becomes a slippery slope.

**Comment by kuien** [comment]
I have no idea on it, if customers don't need it at least till now, let's delay it to 8.4 step.
